packages/terra-clinical-onset-picker/tests/wdio/onset-spec.js

```diff
@@ -370,5 +370,22 @@ viewports.forEach((viewport) => {
 
       Terra.should.validateElement({ axeRules });
     });
+
+    describe('Displays legend when specified', () => {
+      before(() => {
+        browser.url('/#/raw/tests/terra-clinical-onset-picker/clinical-onset-picker/with-legend');
+      });
+
+      Terra.should.validateElement({ axeRules });
+    });
+
+    describe('Hides legend when hide button is clicked', () => {
```

Comment:
The ‘hide button’ is the implementation detail of your test component. I’d create two small stateless test components : one with ‘isLegendHidden’ set, one without and use them here.

Reply:
Done : cf0acae75d625ca596c419b387be98c9365761a5 ,
Added two test components one with ‘isLegendHidden’ set, one without ‘isLegendHidden’.